mlir/lib/Target/LLVMIR/DebugImporter.cpp

```diff
@@ -56,10 +56,14 @@ DIBasicTypeAttr DebugImporter::translateImpl(llvm::DIBasicType *node) {
 DICompileUnitAttr DebugImporter::translateImpl(llvm::DICompileUnit *node) {
   std::optional<DIEmissionKind> emissionKind =
       symbolizeDIEmissionKind(node->getEmissionKind());
+  std::optional<DINameTableKind> nameTableKind = symbolizeDINameTableKind(
+      static_cast<
+          std::underlying_type_t<llvm::DICompileUnit::DebugNameTableKind>>(
```

Comment:
Is this necessary? For DIEmissionKind we seem to use the enum directly without static cast? Or should both use a cast to the underlying type in both cases?

Reply:
Yeah interestingly this NameTableKind is declared an `enum class`, whereas EmissionKind is just an `enum`.